mllib/src/test/scala/org/apache/spark/ml/feature/CountVectorizerSuite.scala

```diff
@@ -183,6 +183,26 @@ class CountVectorizerSuite extends SparkFunSuite with MLlibTestSparkContext
       case Row(features: Vector, expected: Vector) =>
         assert(features ~== expected absTol 1e-14)
     }
+
+    // CountVectorizer test
+    val df2 = sqlContext.createDataFrame(Seq(
```

Comment:
I think @MLnick mentioned this, but you could just use the same DataFrame as above, no need to make a second one